### PR TITLE
add FeatureInfo functionality for Raster Data sources

### PR DESCRIPTION
**Pull request by taulinger**
- display FeatureInfo for Raster Data sources

### CODE REVIEW

File: src/plugins/i18n/featureInfoPlugin.provider.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
+export const provide = (lang) => {
+	switch (lang) {
+
+		case 'en':
+			return {
+				//the first part of the snake_case key should be the name of the related plugin
+				featureInfoPlugin_featureInfoService_exception: 'FeatureInfo could not be retreived'
```
Comment (thiloSchlemmer) [inline src/plugins/i18n/featureInfoPlugin.provider.js:8]: Typo -> retrieved

File: test/plugins/FeatureInfoPlugin.test.js [hunk 2]
```diff
@@ -42,11 +81,158 @@ describe('FeatureInfoPlugin', () => {
...
+				setClick({ coordinate: coordinate, screenCoordinate: [33, 44] });
+
+				setTimeout(() => {
+					expect(store.getState().featureInfo.current).toHaveSize(1);
+					expect(store.getState().featureInfo.current[0].content).toBe('content');
+					expect(store.getState().featureInfo.current[0].title).toBe('title');
+				});
+			});
+
+			it('adds FeatureInfo items taking layerPropeties\' label as title', async () => {
```
Comment (thiloSchlemmer) [inline test/plugins/FeatureInfoPlugin.test.js:115]: typo layerPrope(r)ties

File: test/plugins/FeatureInfoPlugin.test.js [hunk 3]
```diff
@@ -42,11 +81,158 @@ describe('FeatureInfoPlugin', () => {
...
+				spyOn(mapService, 'calcResolution').withArgs(zoom, coordinate).and.returnValue(resolution);
+				const featureInfoServiceSpy = spyOn(featureInfoService, 'get');
+				await instanceUnderTest.register(store);
+
+				setClick({ coordinate: coordinate, screenCoordinate: [33, 44] });
+
+				expect(featureInfoServiceSpy).not.toHaveBeenCalled();
+			});
+
+			it('adds No FeatureInfo items when service returns no result', async () => {
```
Comment (thiloSchlemmer) [inline test/plugins/FeatureInfoPlugin.test.js:172]: typo ...somehow -> 'No'

...if the capitalize was by intention, more a point for a style discussion later on: I used in my tests the pattern, to capitalize the negation , too. 
Should me make this a common rule? 
In this case: "adds NO FeatureInfo items when service returns NO result"

**Review by thiloSchlemmer — DISMISSED** [review]
Nice! Some comments inline.

File: test/plugins/FeatureInfoPlugin.test.js (hunk 3 above)
Comment (taulinger) [reply test/plugins/FeatureInfoPlugin.test.js:172]: good point

**Review by thiloSchlemmer — APPROVED** [review]
THX!

**Pull request merged**